src/libmongoc/doc/api.rst

```diff
@@ -23,6 +23,7 @@ API Reference
    mongoc_client_encryption_datakey_opts_t
    mongoc_client_encryption_rewrap_many_datakey_result_t
    mongoc_client_encryption_encrypt_opts_t
+   mongoc_encrypt_text_opts_t
```

Comment:
As much as I do not like the verbosity, consider renaming for consistency with other explicit encryption API:

```
mongoc_encrypt_text_opts_t            => mongoc_client_encryption_encrypt_text_opts_t
mongoc_encrypt_text_prefix_opts_t     => mongoc_client_encryption_encrypt_text_prefix_opts_t
mongoc_encrypt_text_suffix_opts_t     => mongoc_client_encryption_encrypt_text_suffix_opts_t
mongoc_encrypt_text_substring_opts_t  => mongoc_client_encryption_encrypt_text_substring_opts_t
```

That avoids two naming styles for explicit encryption API. A future change can add shorter aliases for all symbols.